app/services/appointment_notification_service.rb

```diff
@@ -16,17 +16,32 @@ def initialize(appointments:, days_overdue: 3)
 
   def send_after_missed_visit
     eligible_appointments = appointments.eligible_for_reminders(days_overdue: days_overdue)
+    next_messaging_time = Communication.next_messaging_time
 
     eligible_appointments.each do |appointment|
+      # i don't believe this is the best way to control this. combined with the fact that
+      # we grab all appointments with remind_on before today, I suspect this means we're
+      # resending the same reminders because they failed the first time.
+      # leaving it in place for now in an effort to change as little of the current process as possible.
```

Comment:
Yeah, this is where we could do a tiny PR to add some logging here to see if we have repeated old notifications going out and failing everyday.

Reply:
I've made a 1off request that I'm hoping sheds light on that question.